x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -308,6 +312,7 @@ const usePolicyTemplateInitialName = ({
       name: currentIntegrationName,
     });
     setCanFetchIntegration(false);
+    invalidateQueryCache();
```

Comment:
PR is working. But I wonder if this invalidation shouldn't occur prior to the calculation of `packagePolicyListByIntegration`. It seems that our workflow is to obtain the latest list and generate `getMaxPackageName` based on that when mounting. In this case, I would assume that the invalidate hook should be first on this useEffect.

A bit more nit picky but it's usually better for the invalidate action to occur after a user has performed an action, specifically a save in our case. since after the save, we know the list will get longer, prompting the need to invalidate the query. If we still prefer to do it on mount regardless if a successful save has happened or not, it might be better to use `refetch`.

Reply:
Since we don't have access to an `onSubmit` method the best solution for this scenario ended up being to call `refetch` once the page is mounted.